test/test_common/environment.cc

```diff
@@ -145,26 +145,27 @@ std::string TestEnvironment::temporaryFileSubstitute(const std::string& path,
   return temporaryFileSubstitute(path, ParamMap(), port_map, version);
 }
 
-// TODO(htuch): Switch to a single pass variable substitution.
```

Comment:
Should we keep this TODO?

Reply:
I didn't think so - we've mostly deprecated this code.  It's used in legacy tests only and not worth the tie to performance optimize IMO

Reply:
OK SGTM.